submissions/chabuChabu/javaScript/index.js

```diff
@@ -0,0 +1,24 @@
+var productList = [
+   {
+    productName: "Red Tennis Shoes",
+    imageUrl: "../html/images/product-02.jpg",
+    productPrice: "$200",
+   },
+   {
+    productName: "Black Office shoes",
+    imageUrl: "../html/images/product-03.jpg",
+    productPrice: "$350",
+   },
+   {
+    productName: "Red High-heeled shoes",
+    imageUrl: "../html/images/product-04.jpg",
+    productPrice: "$230",
+   }
+]
+console.log(productList[0].productName+" with "+productList[0].imageUrl+" is sold for "+productList[0].productPrice);
```

Comment:
@ChabuChabu You are meant to loop through the array and make an output not one after the other.

Reply:
Code changed